orchestrator/gravity_utils/src/types/checksum.rs

```diff
@@ -0,0 +1,29 @@
+use ethers::utils::hex;
+use sha3::{Digest, Keccak256};
+
+pub fn checksum(address: &str) -> String {
```

Comment:
some ref doc that describes how the checksum is computed or attribution to the original code where it comes from if it was copy-pasted from somewhere?

Reply:
added